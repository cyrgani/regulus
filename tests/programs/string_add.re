assert_eq(+("a", "b"), "ab"),
assert_eq(+(1, 2), 3),
__builtin_print_catch(+("1", 2)),
__builtin_print_catch(+(1, "2")),
assert_eq(+("", ""), ""),
assert_eq(+("", endl()), endl()),
