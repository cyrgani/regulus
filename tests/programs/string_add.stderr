ArithmeticError: Unsupported addition
at <stl:operators>:3:31
at <stl:operators>:25:32
at <stl:operators>:22:19
at <stl:operators>:19:15
at <stl:operators>:16:11
at <stl:operators>:13:19
at programs/string_add.re:3:24
at programs/string_add.re:3:22
ArithmeticError: Unsupported addition
at <stl:operators>:3:31
at <stl:operators>:25:32
at <stl:operators>:22:19
at <stl:operators>:19:15
at <stl:operators>:16:11
at <stl:operators>:13:19
at programs/string_add.re:4:24
at programs/string_add.re:4:22
